FAQ.md

```diff
@@ -115,3 +115,37 @@ Runtime platforms are in our [supported platforms page](https://adoptium.net/sup
 ## How to add a new build pipeline param and associated job configuration?
 
 [This PR](https://github.com/adoptium/temurin-build/pull/2416) demonstrates changes required to add a new build pipeline param and the associated version/platform job configurations for setting the value when needed  (note, the `pipelines/` dir has since been moved to our [jenkins repository](https://github.com/adoptium/ci-jenkins-pipelines)).
+
+## How do I build from a tag(without docker)
+
+The following are the pre-requisites for the the build to be successful
```

Comment:
Need to state what the initial OS environment is? I'd have expected Gcc to be installed? build-essential package typically.

Reply:
So the required minimum Gcc depends on version, see: https://wiki.openjdk.org/display/Build/Supported+Build+Platforms